src/components/scene.js

```diff
@@ -205,12 +239,16 @@ export class Scene extends Component {
                 useDevicePixels={useDevicePixels}
                 style={style}
                 views={new MapView({repeat: true})}
+                // views={views}
                 /* onBeforeRender={this._onBeforeRender} // Not yet
                       onHover={visStateActions.onLayerHover} // Not yet
                       onClick={visStateActions.onLayerClick}*/ // Not yet
-                {...this.props.adapter.getProps(this.deckgl, () => {}, () => {this.forceUpdate()})}
+                {...this.props.adapter.getProps(this.deckgl, () => {}, () => {this.forceUpdate()})} /* @Chris My understanding (from Elisa) is that this is forced because this.deckgl is not defined initially */
```

Comment:
This is likely the onNextFrame callback you and I had gone over in the hubble.gl example onboarding task earlier, it’s appropriate to use forceUpdate.

Reply:
👍 Got it. I was looking up the React Docs and it said to try to avoid forceUpdates and that's why I flagged it in my notes